README.md

```diff
@@ -31,15 +31,11 @@ More instructions are below
 
 ### The docker environment can be set by the commands below.
 
-Running a docker container from a linux system is done with this sequence. At the end, the `dev_into.sh`
-script will get you into the container
+First, you need to (install docker-ce properly)[https://github.com/ApolloAuto/apollo/blob/master/docker/scripts/README.md#install-docker].
```

Comment:
the Markdown syntax should be``[]()``
:-)

Reply:
Oops. Nice catch!